---
title       : "Ancient DNA"
subtitle    : "BIOL14406: Living Systems"
author      : Dr Axel Barlow
job         : "email: axel.barlow@ntu.ac.uk"
framework   : io2012        # {io2012, html5slides, shower, dzslides, ...}
highlighter : highlight.js  # {highlight.js, prettify, highlight}
hitheme     : zenburn      # {zenburn, tomorrow, solarized-dark, ...}
widgets     : []            # {mathjax, quiz, bootstrap}
mode        : selfcontained # {selfcontained, standalone, draft}
knit        : slidify::knit2slides
logo        : ntu-shield.png
biglogo     : NTU_open-graph.png
assets      : {assets: ../../assets}
license     : by-nc-sa
github:
  user: draxelbarlow
  repo: BIOL14406_ancient_DNA
  branch: "gh-pages"
---

```{r setup, include = FALSE}
library(tidyverse)
library(gganimate)
library(RColorBrewer)
library(knitr)
library(ape)
```

<!-- adding bold and italic options -->
<style>
em {
  font-style: italic
}
strong {
  font-weight: bold;
}
</style>

## Ancient DNA

- Mitochondrial phylogenetics
- Ancient DNA
- Sabretooth cats
- Sequencing the North Sea *Homotherium*

```{r, out.width = "45%", dpi = 300, echo = FALSE, fig.align='right'}
include_graphics("./assets/img/smilodon1.png")
```

--- .class #id

## Paijmans et al. 2017

```{r, out.width = "100%", out.height=500, dpi = 300, echo = FALSE}
include_graphics("./assets/img/Paijmans et al. - 2017.pdf")
```

--- .segue .dark 

## Mitochondrial phylogenetics

--- &twocol bg:white

## You've probably seen diagrams like this...

### This is called a `phylogeny`. It represents the evolutionary process

*** =right

```{r, out.width = "80%", fig.width = 3, fig.height = 3, dpi = 600, fig.align = "center", echo = FALSE}

my.tree <- read.tree(text="((((human,chimp),(cat,dog)),(snake,lizard)),fish);")
par(mar=c(1,1,1,1))
plot(my.tree)
```

*** =left

- The phylogeny works like an evolutionary tree
- The tips are species
- The branches show their relationships
- Nodes represents common ancestors, from which new species evolved
- Generally, phylogenies are calculated using **genetic data**

--- .class #id

## Inheritance plus mutation

- Errors in DNA replication (mutations) change the DNA sequence
- Human mutation rate is ~1.1×10−8 mutations per site per generation
- This means you carry ~40 mutations in your 3.6 Gb genome
- These differences are inherited by offspring
- Sequence divergence and time are (approximately) linearly related

### Phylogenetics works in reverse: we observe the DNA sequences and try to work out the tree that generated them

--- .class #id

## DNA sequence alignment

```{r, out.width = "90%", dpi = 300, echo = FALSE, fig.align='center'}
include_graphics("./assets/img/DNA_alingment.png")
```

--- .class #id

## Mitochondrial DNA is great for phylogenetics!

```{r, out.width = "70%", dpi = 300, echo = FALSE, fig.align='center'}
include_graphics("./assets/img/Mitochondrion_mini.svg")
```

--- &twocol

## Mitochondrial DNA is great for phylogenetics!

*** =right

```{r, out.width = "100%", dpi = 300, echo = FALSE, fig.align='center'}
include_graphics("./assets/img/Map_of_the_human_mitochondrial_genome.svg")
```

*Emmanuel Douzery, CC BY-SA 4.0*

*** =left

- Mitochondrial genome ~16 kb
- 13 protein-coding genes, 2 rRNAs and 22 tRNAs
- High mutation rate
- No recombination
- High copy number per cell
- Maternally inherited

--- .class bg:white

```{r, out.width = "95%", dpi = 300, echo = FALSE, fig.align='center'}
include_graphics("./assets/img/mtDNA_tree.svg")
```

--- .segue .dark 

## Ancient DNA

--- .class #id

## Ancient DNA is just old DNA

```{r, out.width = "80%", dpi = 300, echo = FALSE, fig.align='center'}
include_graphics("./assets/img/samples.png")
```

--- .class #id

## Some samples don't fit in the lab!

```{r, out.width = "65%", dpi = 300, echo = FALSE, fig.align='center'}
include_graphics("./assets/img/big_bone.jpg")
```

--- &vcenter

## Timeline of ancient DNA

```{r, out.width = "100%", dpi = 300, echo = FALSE, fig.align='center'}
include_graphics("./assets/img/timeline.svg")
```

--- .class #id

## Study species

```{r, out.width = "90%", dpi = 300, echo = FALSE, fig.align='center'}
include_graphics("./assets/img/extinct.png")
```

*Glyptodont, WolfmanSF, CC BY_SA 3.0*

--- .class #id

## What are the challenges?

```{r, out.width = "80%", dpi = 300, echo = FALSE, fig.align='center'}
include_graphics("./assets/img/workflow.svg")
```

--- .class #id

## Ancient DNA work is contamination sensitive

```{r, out.width = "80%", dpi = 300, echo = FALSE, fig.align='center'}
include_graphics("./assets/img/sina_lab.jpg")
```

--- .class #id

## Ancient DNA work is contamination sensitive

```{r, out.width = "50%", dpi = 300, echo = FALSE, fig.align='center'}
include_graphics("./assets/img/sinalab2.JPG")
```

--- &vcenter

## But the samples are already highly contaminated

```{r, out.width = "100%", dpi = 300, echo = FALSE, fig.align='center'}
include_graphics("./assets/img/endo.svg")
```

--- .segue .dark 

## Sabretooth cats

--- &twocol

## There were two genera of sabretooth cats

- **A genus us a group of closely related species. It's the first part of the scientific name**
- **e.g. Homo sapiens**

*** =left

### Smilodon

- The most commonly known group
- Massive canines
- > 400 kg and 120 cm shoulder height
- N and S America
- 3 species, *S. gracilis, S. populator, S. fatalis*
- Extinction 10 ka

*** =right

```{r, out.width = "100%", dpi = 300, echo = FALSE, fig.align='center'}
include_graphics("./assets/img/smilodon_skull.jpg")
```

--- &twocol

## There were two genera of sabretooth cats

- **A genus us a group of closely related species. It's the first part of the scientific name**
- **e.g. *Homo sapiens*)**

*** =left

### Homotherium

- Less known group
- Also known as scimitar-toothed cats
- Flat, serrated canines
- ~ 200 kg and 110 cm shoulder height
- Europe, Africa, N and S America
- Europe: *H. latidens*, extinction 300 ka
- N. America: *H. serum*, extinction 12 ka

*** =right

```{r, out.width = "100%", dpi = 300, echo = FALSE, fig.align='center'}
include_graphics("./assets/img/Homotherium_serum.jpg")
```

*Sergiodlarosa, CC BY-SA 3.0*

--- .class #id

## *Homotherium* skull

```{r, out.width = "90%", dpi = 300, echo = FALSE, fig.align='center'}
include_graphics("./assets/img/Homotherium_crenatidens_skull_45.jpg")
```

*Ghedoghedo, CC BY-SA 4.0*

--- &twocol

## Fishing for fossils

*** =left

- Britain connected to mainland Europe by an area called **Doggerland**
- Rising sea levels 6-7 ka flooded the area, making Britain an island

```{r, out.width = "90%", dpi = 300, echo = FALSE, fig.align='center'}
include_graphics("./assets/img/ship-2262650.jpg")
```

*** =right

```{r, out.width = "90%", dpi = 300, echo = FALSE, fig.align='center'}
include_graphics("./assets/img/Doggerland.svg")
```

*Max Naylor, CC BY-SA 3.0*

--- &vcenter

## 16th March 2000, something surprising turned up...

```{r, out.width = "100%", dpi = 300, echo = FALSE, fig.align='center'}
include_graphics("./assets/img/NS_Homo.png")
```

### This didn't look like a 300 ka fossil

--- .class #id

## Analysis of the Dutch North Sea *Homotherium*

### Dating

 >- The bone was radiocarbon dated at 31,300 ± 400!
 >- This was extraordinary, so the dating was repeated:
  + 31,300 ± 400
  + 26,900 ± 400
  + 26,700 ± 240
  + 28,100 ± 220
  + 27,650 ± 280

>- The first Late Pleistocene European *Homotherium*

--- .class #id

## Analysis of the Dutch North Sea *Homotherium*

### Ancient DNA

```{r, out.width = "50%", dpi = 300, echo = FALSE, fig.align='left'}
include_graphics("./assets/img/hofreiter_02c.png")
```

- A sample was taken in 2006 by Prof. Michi Hofreiter
- Some tiny sequences were recovered, but they were too short to provide conclusive results...

--- .segue .dark 

## Sequencing the North Sea *Homotherium*

--- &twocol

## By 2014 the technology had improved substantially

*** =left

- Widespread adoption of Next Generation Sequencing by the aDNA community
- Improved DNA extraction methods
- Improved library preparation methods
- A targeted sequencing technique called **DNA hybridisation capture**

*** =right

```{r, out.width = "70%", dpi = 300, echo = FALSE, fig.align='center'}
include_graphics("./assets/img/me_cropped.jpg")
```

- Michi gave the sample to his PhD student, Johanna Paijmans, to try again

--- &vcenter

## Initial sequencing showed extremely high contamination

```{r, out.width = "100%", dpi = 300, echo = FALSE, fig.align='center'}
include_graphics("./assets/img/endo.svg")
```

--- &twocol

## DNA hybridisation capture

### "Fishes" mitochondrial fragments from the contaminated ancient DNA

*** =left

- DNA has 2 strands, arrange in a double helix
- It can be heat denatured
- When cooled, the single strands will stick (hybridise) to strands with a similar sequence
- Manufacture "bait" sequences that match your target region
- Use to capture these sequences from the contaminants

*** =right

```{r, out.width = "100%", dpi = 300, echo = FALSE}
include_graphics("./assets/img/Dna-base-flipping.svg")
```

*Magladem96, CC BY-SA 3.0*

--- &vcenter

## DNA hybridisation capture

```{r, out.width = "90%", dpi = 300, echo = FALSE}
include_graphics("./assets/img/hybr.svg")
```

### This only works if you know the sequence in advance

--- &twocol

## Meanwhile a Danish group were sequencing North American cave lion mtDNA

*** =right

```{r, out.width = "100%", dpi = 300, echo = FALSE, fig.align='center'}
include_graphics("./assets/img/NA_Homo.png")
```

*** =left

- Sequence analysis showed it was actually a *Homotherium*
- They provided the sequence and Johanna ordered the hybridisation cature baits

--- &vcenter

## The hybridisation capture worked!

```{r, out.width = "100%", dpi = 300, echo = FALSE, fig.align='center'}
include_graphics("./assets/img/endo2.svg")
```

--- .class #id

## Phylogenetic analysis of sabretooth cats

```{r, out.width = "80%", dpi = 300, echo = FALSE, fig.align='center'}
include_graphics("./assets/img/sabre_tree_NODATE.svg")
```

--- .class #id

## The final step: molecular dating

- Molecular phylogenies are scaled to genetic divergence
- We know genetic divergence and time are linearly related

### If we know some of the divergence times, we can scale the tree to time

--- .class #id

## Phylogenetic analysis of sabretooth cats

```{r, out.width = "80%", dpi = 300, echo = FALSE, fig.align='center'}
include_graphics("./assets/img/sabre_tree.svg")
```

--- .class #id

## Molecular dating of sabretooth cats

- Sabretooths divergence from living cats 20 Ma
- *Homotherium* and *Smilodon* were more diverged from one another than any living cats
- A huge diversity was lost with the extinction of the sabretooths
- North American and European *Homotherium* were genetically similar
- We recommended they be treated as a single species, *H. latidens*

--- .class #id

## Paijmans et al. 2017

```{r, out.width = "100%", out.height=500, dpi = 300, echo = FALSE}
include_graphics("./assets/img/Paijmans et al. - 2017.pdf")
```

--- &thankyou

## Ancient DNA

**An exciting and dynamic field driven by technological advances**

